### ISSUE TITLE
"Chat Interface" block

### ISSUE THREAD

**Issue by thislooksfun** [OpenMods/OpenBlocks#392]
A couple days ago I came up with an idea for a "chat interface" block while watching vswe and Hanse00 struggle while making the second modded rube goldberg machine. Basically, it would have a chat log in it's GUI, just a text list, which would be added to by interacting with a block. For example, right-clicking an Ars Magica 2 mana battery with the Magitech's Staff. Also, it could be interacted with using ComputerCraft. I don't know how to do that part, but for the rest of it, I have some simple example code made. It doesn't fit within any of my mods, and so I thought what better place to send a random block idea. :P
TL; DR: It's an Autonomous Activator that saves chat messages that would otherwise be lost, which can then be read straight from the GUI, or accessed in a table from a CC computer/turtle.

Here's the code:
https://www.dropbox.com/s/2r9a4zbtj8d5zel/Chat%20interface.zip
